tempesta_fw/http.c

```diff
@@ -786,20 +1369,13 @@ tfw_http_req_cache_service(TfwHttpReq *req, TfwHttpResp *resp)
 {
 	if (tfw_http_adjust_resp(resp, req))
 		goto resp_err;
-
-	if (tfw_cli_conn_send(req->conn, (TfwMsg *)resp))
-		goto resp_err;
-
+	tfw_http_resp_fwd(req, resp);
```

Comment:
We should use lightweight logic here as previously, see comment for `tfw_http_req_process()`.